docs/submitting-clinical-data.md

```diff
@@ -14,17 +14,15 @@ Clinical data consists of all supporting data about your program's donors and th
 1. Upon uploading, if there are any errors in your file they will be displayed within the Clinical Submission workspace. The error report will also be available for download. You must fix all of the errors that are listed within your file(s) and then reupload them.
 1. Valid files will be available for review within the file preview tables. You can review the file data for each data type using by using the tabs on the left of the preview area.
```

Comment:
remove the extra "using" before "by using"

Reply:
i dont see an extra using?

Reply:
found it :p